app/styles/component/_featured-content.scss

```diff
@@ -2,38 +2,54 @@
 .featured-content {
 	$featured-content-caption-line-height: 32px;
 
+	height: 56.25vw;
 	margin-bottom: $featured-content-margin-bottom;
 	position: relative;
 	width: 100%;
 
-	.chevron {
+	> .wds-icon {
 		fill: $white;
 		height: 100%;
 		position: absolute;
 		stroke: rgba(0, 0, 0, .5);
 		stroke-width: .25;
 		top: 0;
+		z-index: 1;
```

Comment:
How about using a $z-1 or defining a $z-index-featured-content-icon in wikia-variables.scss?